### PR TITLE
[bugfix](paimon)fix paimon testcases

### PR DESCRIPTION
**Pull request by wuwenchi**
## Proposed changes

1. set default timezone
2. not supported `char` type to pushdown

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by wuwenchi** [comment]
run buildall

File: be/src/vec/exec/format/orc/vorc_reader.cpp [hunk 1]
```diff
@@ -524,8 +525,20 @@ std::tuple<bool, orc::Literal> convert_to_orc_literal(const orc::Type* type, con
 
 template <PrimitiveType primitive_type>
 std::vector<OrcPredicate> value_range_to_predicate(
-        const ColumnValueRange<primitive_type>& col_val_range, const orc::Type* type) {
+        const ColumnValueRange<primitive_type>& col_val_range, const orc::Type* type,
+        std::vector<orc::TypeKind>* cannot_predict_types) {
```
Comment (morningman) [inline be/src/vec/exec/format/orc/vorc_reader.cpp:529]:
```suggestion
        std::vector<orc::TypeKind>* unsupported_pushdown_type) {
```

File: be/src/vec/exec/scan/vfile_scanner.cpp [hunk 2]
```diff
@@ -785,9 +785,17 @@ Status VFileScanner::_get_next_reader() {
             break;
         }
         case TFileFormatType::FORMAT_PARQUET: {
+            static const cctz::time_zone utc0 = cctz::utc_time_zone();
+            cctz::time_zone* tz;
+            if (range.__isset.table_format_params &&
+                range.table_format_params.table_format_type == "paimon") {
+                tz = const_cast<cctz::time_zone*>(&utc0);
```
Comment (morningman) [inline be/src/vec/exec/scan/vfile_scanner.cpp:792]: Why paimon use utc?
add comment in code

**Comment by wuwenchi** [comment]
run buildall

File: be/src/vec/exec/scan/vfile_scanner.cpp (hunk 2 above)
Comment (wuwenchi) [reply be/src/vec/exec/scan/vfile_scanner.cpp:795]: done

File: be/src/vec/exec/format/orc/vorc_reader.cpp (hunk 1 above)
Comment (wuwenchi) [reply be/src/vec/exec/format/orc/vorc_reader.cpp]: done

**Comment by wuwenchi** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Comment by wuwenchi** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by zddr — APPROVED** [review]

**Pull request merged**